backend/User/views.py

```diff
@@ -95,7 +138,7 @@ def post(self, request):
 			login(request, user)
 			# set additional session data if necessary
 			request.session['is_authenticated'] = True
```

Comment:
In order to use the localStorage in the UI to get for example logged in users id in order to make new requests for the user by its id, we return the user from the login!

Reply:
This is just for the logged in user? because you return the friend already, right?

on a separate note. what url should I display to the user, when they are on a friends page? Do we want them to even navigate to their friends via the url bar? I will make an issue for this

Reply:
Yup, this is for the logged in user, friends will get their pictures on a different route.